### ISSUE TITLE
Typescript Standard linting broken in v5.0.0

### ISSUE THREAD

**Issue by gmaiwald** [super-linter/super-linter#4445]
## Describe the bug

I upgraded Super Linter from version 4.9.0 to version 5.0.0. Since then the TS Standard Linting fails. On each file it processes it prints the following error message:

```
------
ts-standard: Standard for TypeScript! (https://github.com/standard/ts-standard)
  /tmp/lint/bin/super-linter-ts-trouble.ts:0:0: Parsing error: ESLint was configured to run on `<tsconfigRootDir>/bin/super-linter-ts-trouble.ts` using `parserOptions.project`: /action/lib/.automation/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file (null)
------
```

## Expected behavior

The hints on the referred url don't work for me. If I run the linter 4.9.0, it complains, but the files are processed anyway:

## Steps to Reproduce

```
git init
cdk init app --language=typescript
git add -A
git commit -m 'initial commit'
docker run -e RUN_LOCAL=true -e VALIDATE_TYPESCRIPT_STANDARD=true -v .:/tmp/lint github/super-linter:v5.0.0
```

If I run the linter 4.9.0, it complains, but the files are processed anyway:
```
docker run -e RUN_LOCAL=true -e VALIDATE_TYPESCRIPT_STANDARD=true -v .:/tmp/lint github/super-linter:v4.9.0
```

Do you have any idea or even a solution for this problem? Thanks in advance

**Comment by zkoppert** [comment]
Looks like the TS linter is unable to run on a file by file basis and needs to be able to be aware of all included files in order to perform type-aware linting. We could turn this off by default and it may solve the problem.

**Comment by zkoppert** [comment]
Is there a public repository I can test this out on? My cdk installation is not working properly at the moment.

**Comment by gmaiwald** [comment]
You can test it out on: https://github.com/gmaiwald/super-linter-ts-trouble
Hope this helps.

**Comment by zkoppert** [comment]
I ran this with the latest version `super-linter/super-linter` v5 tag and it is passing without error. You can see the log for it at https://github.com/zkoppert/super-linter-ts-trouble/actions/runs/5479984662/jobs/9982555873 . Can you update to the latest v5 and try.

**Comment by gmaiwald** [comment]
Do you already have new insights?

**Comment by zkoppert** [comment]
Reproduced the issue at https://github.com/zkoppert/super-linter-ts-trouble/actions/runs/5480167362/jobs/9982985722

**Comment by zkoppert** [comment]
Created a PR to see if I can remove type-checking by default. Testing that now to see if my change in fact does that and if it resolves the reproduced error.

**Comment by zkoppert** [comment]
Ran into an issue testing this out because I'm on a arm based mac so I can't create the docker image to test this out. Going to try to work on that problem and come back to this once unblocked. Open to alternatives if anyone had got any ideas.

**Comment by agustindelmonti** [comment]
Any updates on this issue? I am experiencing the same error :(

**Comment by zkoppert** [comment]
@agustindelmonti Unfortunately I don't have an update yet. Fixing the docker issues are estimated as a large effort so this will likely take a while.

**Comment by gmaiwald** [comment]
Hello, I saw that ticket #4445 is closed now. Does it mean the bug is fixed? 
I could reproduce error with v5.3.0

**Comment by zkoppert** [comment]
That was an attempt solve this issue, but as you mentioned the issue is still there. Looks like we will need to come up with another way to solve this issue. 🤔

**Comment by ferrarimarco** [comment]
@gmaiwald can you please help us reproduce the issue with a simple typescript file and related configuration file?

Thanks!

**Comment by ferrarimarco** [comment]
I tried running this example as a super-linter test case, but I wasn't able to reproduce the issue.

> The source at https://github.com/zkoppert/super-linter-ts-trouble/actions/runs/5480167362/jobs/9982985722

Closing for now. Please, feel free to open a new issue or comment on this one if this happens again. Thanks!

**Comment by kshitij1411** [comment]
Still seeing this error:

ts-standard: Standard for TypeScript! (https://github.com/standard/ts-standard)
  /tmp/lint/sample-codes/sample.ts:0:0: Parsing error: ESLint was configured to run on `<tsconfigRootDir>/sample-codes/sample.ts` using `parserOptions.project`: /action/lib/.automation/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file (null)

Is this issue resolved?

Ran it with below command:

docker run -e RUN_LOCAL=true \
  -e VALIDATE_ALL_CODEBASE=false \
  -v $(pwd):/tmp/lint \
  github/super-linter:latest
